apps/trading-e2e/src/integration/withdrawals.feature

```diff
@@ -1,57 +1,49 @@
-@ignore
-Feature: Withdrawals
+Feature: Withdrawals to eth wallet
 
-  Scenario: Can prepare a withdrawal
-    Given I am on withdraw page
-    And I connect eth wallet
-    And I connect vega wallet
-    And I can see the withdrawals warning info
-    And I can see the from ethereum address field is automatically populated with current connected key
-    And I select asset 'tBTC'
-    And I input '100' tokens to withdraw
-    And the withdraw button is enabled
-    When I click the withdraw button
-    Then The vega transaction is sent
-    Then The Ethereum transaction is triggered
+  Background:
+    Given I navigate to withdrawal page
+    And I connect to Vega Wallet
 
-  Scenario: Form field validation if fields are incomplete
-    Given I am on withdraw page
-    And I connect eth wallet
-    And I connect vega wallet
-    When I click the withdraw button
-    Then I can see validation errors on incomplete fields
+  Scenario: Succesfull withdrawal
+    When I enter the following details in withdrawal form
+      | asset  | tEURO TEST |
```

Comment:
follow up (non-blocking): probably want to find these by assetId and have it configurable